spec/spec.yaml

```diff
@@ -8,7 +8,17 @@ python3:
   entrypoint: 'python3'
   filename:   'main.py'
 
+rust:
+  image:      'rust:alpine'
+  cmd:        'rustc main.rs && ./main'
```

Comment:
Why don't you just use `cargo run`?

Reply:
Cargo run would require a cargo project, am I right? That would be not suitable for the docker provider. ^^

Reply:
Right. With rustc you can specifically target a single file

Reply:
yes you can, but cargo does some additional optimizations afaik. And additional dependencies would be supported...
Maybe that is a feature to implement later on

Reply:
I think dependency usage will not be supported that soon because that would require network access for the runner containers.

Network access is currently disabled in the sandboxes to prohibit the usage of this service as botnet node or something similar.